ui/window.go

```diff
@@ -185,6 +199,10 @@ func (win *Window) Loop(shutdown chan int) {
 				go func() {
 					win.keyEvents <- &evt
 				}()
+			case system.ClipboardEvent:
+				go func() {
+					win.clipboardPasteText <- evt.Text
```

Comment:
When this event is triggered by calling window.ReadClipBoard, the text should be sent back to the same channel with the third type. It could be named "Paste". All Editors would listen to the same channel and they would receive the signal when it is sent here. But only the focused editor would have the text pasted in its field.